### DISCUSSION TITLE
Question about internationalization

### DISCUSSION THREAD

**Discussion by JonasCir** [squidfunk/mkdocs-material#2679] [General]
Hi there,
first of all thank you for creating this awesome project! :)

I was trying to setup multi language support for a site I'm maintaining as described in the [docs](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/):

- This is the [repo](https://github.com/hzi-braunschweig/SORMAS-Glossary/)
- Deployment is done via GitHub action to the corresponding GitHub [page](https://hzi-braunschweig.github.io/SORMAS-Glossary/en/)

I added the site language selector to [mkdocs.yml](https://github.com/hzi-braunschweig/SORMAS-Glossary/blob/master/mkdocs.yml) in my repo root. The language selector shows of and if a I select a different language, I navigate to the correct index file.

The following is not working for me:
1. If I select `French` as language, the translation does not change (i.e., `"search.placeholder": "Rechercher"` is not set)
1. The navigation on the left side shows both language folders in a tree view

Can you tell me if this is a bug or my setup is just wrong? If the latter is the case, I'm happy to contribute to the documentation to make things clearer. If it's a bug, I'm going to raise an issue.

Thank you in advance!

**Comment by JonasCir** [comment]
Ok I just found https://github.com/squidfunk/mkdocs-material/discussions/2346 , I wonder why this did not show up in my google search :D

But then I wonder how I'm going to deploy it to GitHub pages correctly? Since I need to run a  `mkdocs build ` for each language the action showed [here](https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions) won't work anymore

**Reply by squidfunk** [reply]
I suggest you ask this question in #2346. I haven't yet worked out a canonical way to deploy multi-language documentation, but maybe the OP of #2346 can shed some light on this topic.
